### PR TITLE
fix: refactor vfolder storage proxy components

### PR DESCRIPTION
**Pull request by lizable**
### Description
---------

This PR resolves four minor problems:
- column width of storage-proxy list
- font size of text in lablup-piechart
- move utilizable functions ( `_markIfUnlimited()`) to `backendai-common-utils` component.
- use proper unit conversion
    - MB, GB, TB,... are only used when converting bytes to decimal, not binary.

### CODE REVIEW

File: resources/i18n/ko.json [hunk 1]
```diff
@@ -422,6 +422,7 @@
     "Create": "생성",
     "Update": "갱신",
     "Copy": "복사",
+    "FolderAlreadyExists": "폴더 명이 이미 존재합니다.",
```
Comment (inureyes) [inline resources/i18n/ko.json:467]: 폴더 명 → 폴더 이름.

File: src/components/backend-ai-agent-list.ts [hunk 2]
```diff
@@ -782,8 +752,14 @@ export default class BackendAIAgentList extends BackendAIPage {
...
 
+  /**
+   * Convert the value byte to MB.
+   *
+   * @param {number} value
+   * @return {number} converted value from byte to MB.
+   */
   _bytesToMB(value) {
-    return Number(value / (1024 * 1024)).toFixed(1);
+    return Math.floor(value / 1000000);
```
Comment (inureyes) [inline src/components/backend-ai-agent-list.ts]: IT should not be 1000000 but be 1048576.

File: src/components/backend-ai-session-list.ts [hunk 3]
```diff
@@ -295,6 +295,16 @@ export default class BackendAiSessionList extends BackendAIPage {
...
   }
 
+  /**
+   * Convert the value byte to MB.
+   *
+   * @param {number} value
+   * @return {number} converted value from byte to MB.
+   */
+  _bytesToMB(value) {
+    return Math.floor(value / 1000000);
```
Comment (inureyes) [inline src/components/backend-ai-session-list.ts:393]: It should be 1048576.

File: src/components/backend-ai-session-view.ts [hunk 4]
```diff
@@ -300,12 +300,14 @@ export default class BackendAiSessionView extends BackendAIPage {
...
-
+  /**
+   * Convert the value byte to MB.
+   *
+   * @param {number} value
+   * @return {number} converted value from byte to MB.
+   */
   _bytesToMB(value) {
-    return Number(value / (1024 * 1024)).toFixed(1);
+    return Math.floor(value / 1000000);
```
Comment (inureyes) [inline src/components/backend-ai-session-view.ts:321]: It should be 1048576.

File: src/components/backend-ai-credential-view.ts [hunk 5]
```diff
@@ -991,7 +991,7 @@ export default class BackendAICredentialView extends BackendAIPage {
     }
   }
 
-  _gBToByte(value: number = 0) {
+  _gBToByte(value = 0) {
```
Comment (inureyes) [inline src/components/backend-ai-credential-view.ts:1000]: Reason to remove type annotation?

**Comment by inureyes** [comment]
@lizable Let's reduce confusion here.

 * For memory: byte to MB → 1048576 (actually it is MiB but is widely used MB for memory)
 * For storage: byte to MB → 1000000 (MiB)

I suggest to make both byteToMiB and byteToMB and apply to storage / memory view differently :).

**Review by inureyes — CHANGES_REQUESTED** [review]
@lizable Let's reduce confusion here.

* For memory: byte to MB → 1048576 (actually it is MiB but is widely used MB for memory)
* For storage: byte to MB → 1000000 (MiB)

I suggest to make both byteToMiB and byteToMB and apply to storage / memory view differently :).

**Comment by lizable** [comment]
Closing this issue since it's been obsolete. We don't use piechart in storage-proxy component.

**Pull request closed without merging**